docs/1-Getting-Started/3-How-it-works.md

```diff
@@ -1,57 +1,111 @@
-**Kool** works with Docker and Docker Compose under the hood to power up small, fast and reproducible local development environments. **Kool** provides simple, no-brainer ease-of-use while keeping all the power and flexibility of Docker and Docker Compose. **Kool** makes using Docker super easy without losing any control, ensuring you remain fully in charge of every detail related to your Docker environments.
+**Kool** is a suite of open source dev tools that level up the way you build and deploy modern, cloud native web applications. By using Kool, you'll get a better development environment and a better development workflow, which adds up to a better developer experience.
```

Comment:
Maybe add emphasis to ``**developer experience**`

Reply:
@fabriciojs - I bolded "a better developer experience" (see https://github.com/kool-dev/kool/pull/362/commits/d29984e3a6b2598fb0eafb4caae556adae6a3e7e).